docs/1.getting-started/6.server-directory.md

```diff
@@ -0,0 +1,53 @@
+---
+navigation.icon: uil:laptop-connection
+description: Using server directory to create middlewares, plugins and API easily.
+---
+
+# Server Directory
+
+Nuxt scans files under server directory to create middlewares, plugins and API routes to create features to use inside components or pages.
+
+Each file under these folders must return a function defined with `defineEventHandler()`.
+
+## Server Routes
+
+Anything under `~/server/api` folder will create API routes to fetch data within API route.
+
+```ts [server/api/hello.ts]
+export default defineEventHandler(() => {
+    return {
+      greetings: 'traveler'
+    }
+})
+```
+
+```vue [components/Greet.vue]
+<script setup lang="ts">
+const {data} = await useFetch('/api/hello')
+</script>
```

Comment:
```suggestion
</script>

<template>
  <div>
     {{ data }}
   </div>
</template>
```

I think that added a full example could help people to copy and paste to understand usage.